app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology.</p>
+<p>In an increasingly digital world, you could inspire pupils to look at a future role in technology. With roles in technology making up 14% of all job opportunities in the UK.</p>
```

Comment:
@caitlin-reidy do we need to say technology after the 1st mention. Could we just say tech as that's how most people would think of it as jobs in tech? (asking as a question!)